images/runtime/python/3.10/buster.Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG DEBIAN_FLAVOR
```

Comment:
Hi @qianz2, will we be adding a base-image-python pipeline? We might need to add a YAML file similar to: https://github.com/microsoft/Oryx/blob/main/vsts/pipelines/baseImages/php.yml

After it goes through the base-image-python pipeline, the image in the similar format would be generated: `mcr.microsoft.com/oryx/python:3.9-20211214.1`

Reply:
Thanks for reminding, I'll add a base image python pipeline in another PR.